specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -300,6 +300,88 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/listresourcepreparationerrors": {
+      "post": {
+        "tags": [
+          "ProxyOperations"
+        ],
+        "operationId": "ListResourcePreparationErrors",
+        "consumes": [],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
```

Comment:
For these parameters, actually we should have the definitions for at least `subscriptionId` and `resourceGroupName` (for instance [here](https://github.com/Azure/azure-rest-api-specs/blob/39e2bfa457e58e57efed3fbbce5ffba87ac24a12/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json#L85)), therefore could we just reuse those definitions instead of defining these again here?

Reply:
Thanks, I will fix it.